src/sql/src/plan/query.rs

```diff
@@ -1518,18 +1521,11 @@ fn plan_view_select(
     s: &Select<Aug>,
     order_by_exprs: &[OrderByExpr<Aug>],
 ) -> Result<SelectPlan, PlanError> {
-    let Select {
-        distinct,
-        projection,
-        from,
-        selection,
-        group_by,
-        having,
-        options,
-    } = s;
+    let mut s = s.clone();
```

Comment:
If we're immediately cloning, we should just request an owned value from a pedantic perspective; don't know that there's much consequence given the call stack, but has cleaner semantics.